src/js/app.js

```diff
@@ -3,6 +3,7 @@ var blocktrail = angular.module("blocktrail.wallet", [
     "ui.bootstrap",
     "ui.bootstrap.dropdown",
     "ui.bootstrap.pagination",
+    "ui.bootstrap.tabs",
```

Comment:
afaik you're no longer using this right?

Reply:
true